spyderlib/cli_options.py

```diff
@@ -39,5 +40,8 @@ def get_options():
     parser.add_option('--profile', action='store_true', default=False,
                       help="Profile mode (internal test, "
                            "not related with Python profiling)")
+    parser.add_option('--travis', dest="travis", action='store_true',
+                      default=False,
+                      help="Run a Qtimer to stop the GUI app in travis-ci")
```

Comment:
What if instead of adding this option to the main interface, we add it to `boostrap.py`?

I think it makes more sense there, don't you think?